unitycatalog-ai/src/ucai/core/client.py

```diff
@@ -42,6 +45,23 @@ def __init__(self, client: Any = None, **kwargs: Any) -> None:  # noqa: ARG002
     @abstractmethod
     def create_function(self, *args: Any, **kwargs: Any) -> Any:
         """Create a function"""
+    
+    @abstractmethod
+    def create_python_function(
+        self, *, func: Callable[..., Any], catalog: str, schema: str, replace: bool = False
+    ) -> "FunctionInfo":
```

Comment:
Can we avoid importing FunctionInfo here? As this is used by OSS client as well, we don't want to introduce databricks dependency in general :)

Reply:
Yep! Sounds good!